optuna/storages/_rdb/alembic/versions/v3.0.0.a.py

```diff
@@ -0,0 +1,235 @@
+"""unify existing distributions to {int,float} distribution
+
+Revision ID: v3.0.0.a
+Revises: v2.6.0.a
+Create Date: 2021-11-21 23:48:42.424430
+
+"""
+from typing import Any
+from typing import List
+
+from alembic import op
+from sqlalchemy import Column
+from sqlalchemy import DateTime
+from sqlalchemy import Enum
+from sqlalchemy import Float
+from sqlalchemy import ForeignKey
+from sqlalchemy import Integer
+from sqlalchemy import orm
+from sqlalchemy import String
+from sqlalchemy import Text
+from sqlalchemy import UniqueConstraint
+from sqlalchemy.engine.reflection import Inspector
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.ext.declarative import declarative_base
+
+from optuna.distributions import BaseDistribution
+from optuna.distributions import DiscreteUniformDistribution
+from optuna.distributions import distribution_to_json
+from optuna.distributions import FloatDistribution
+from optuna.distributions import IntDistribution
+from optuna.distributions import IntLogUniformDistribution
+from optuna.distributions import IntUniformDistribution
+from optuna.distributions import json_to_distribution
+from optuna.distributions import LogUniformDistribution
+from optuna.distributions import UniformDistribution
+from optuna.trial import TrialState
+
+
+# revision identifiers, used by Alembic.
+revision = "v3.0.0.a"
+down_revision = "v2.6.0.a"
+branch_labels = None
+depends_on = None
+
+MAX_INDEXED_STRING_LENGTH = 512
+BATCH_SIZE = 5000
```

Comment:
Do you have any reasons for the value of 5000?

Reply:
No strong evidence here. Documentation says for example 1000 for BATCH_SIZE empirically suite for large amount of records. So it may be sufficient to set 1000.

(I'm thinking that there could be users who have several thousands (1,000~3,000) of trial_params. For such case, refetching would happen on BATCH_SIZE=1,000. So I set 5,000 to aim to avoid refetching for most of use-cases.)